English (US & GB) localization overhaul WIP

EddyMRA opened the pull request:
Greatly improves English localization by rewriting many of the verbose sentences likely machine-translated from a different language. I attempted to use WoW jargon where necessary.

This is a WIP; I got up to the Hunter strategy section about halfway through each localization Lua file.

EddyMRA committed:
English (US & GB) localization overhaul WIP

Greatly improves English localization by rewriting many of the verbose sentences likely machine-translated from a different language. I attempted to use WoW jargon where necessary.

This is a WIP; I got up to the Hunter strategy section about halfway through each localization Lua file.

EddyMRA committed:
English (US & GB) localization overhaul WIP

Greatly improves English localization by rewriting many of the verbose sentences likely machine-translated from a different language. I attempted to use WoW jargon where necessary.

This is a WIP; I got up to the Hunter strategy section about halfway through each localization Lua file.

EddyMRA committed:
enUS and enGB: fixed syntax errors

Fixed missing operators and used British spellings of certain words in the enGB localization file

Wishmaster117 commented:
Hi,
Thanks for your PR, since Macx-Lio is taking some rest, cas you also make it in this provisory repo : https://github.com/Wishmaster117/Multibot_Waiting_his_creator, withou deleting this one, so when Macx-Lio will be back he can merge it to here?

Thanks and have fun :)

EddyMRA committed:
English localization (enUS and enGB) overhail: Revision 1 completed

Added color formatting where necessary. All English lines, including in commented-out sections, have been fully overhauled.

EddyMRA committed:
Minor fixes

EddyMRA committed:
Fixed more syntax errors

EddyMRA committed:
Synced MultiBot.Lua from

Synced from https://github.com/Wishmaster117/Multibot_Waiting_his_creator
